index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="asset/style/main.css" />
+    <!-- Google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Profsain Husseini</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a class="navMenu" href="#">Portfolio</a>
+        <a class="navMenu" href="#">About</a>
+        <a class="navMenu" href="#">Contact</a>
+      </nav>
```

Comment:
- [ ] Kindly change the header of your application to match the Figma design you chose:

## Your header
![your_header](https://user-images.githubusercontent.com/69858269/164327291-ec026f1e-2d57-4fb9-8e05-9d27e7f06e81.PNG)
  
## Figma Header
![figma_header](https://user-images.githubusercontent.com/69858269/164327302-42824784-3749-4256-af57-610c257d1387.PNG)
 
Remember that the logo/title on the left must be an `<a>` tag redirecting to the website itself and the hamburger button must have a `<nav>` tag around it. The logo/title must not be inside the `<nav>` tag, but it must be inside the header tag